### PR TITLE
feat(WebexMeeting): add controls prop to meeting component

### PR DESCRIPTION
**Pull request by karinasigartau0798**
In this PR it was added the controls prop in order to be able to pass a custom list of controls to WebexMeetingControlBar component coming from above. 
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-270964

### CODE REVIEW

File: src/components/WebexMeeting/WebexMeeting.jsx [hunk 1]
```diff
@@ -113,13 +115,15 @@ export default function WebexMeeting({
...
   className: PropTypes.string,
+  controls: PropTypes.func,
   logo: PropTypes.node,
   meetingID: PropTypes.string,
   style: PropTypes.shape(),
 };
 
 WebexMeeting.defaultProps = {
   className: '',
+  controls: undefined,
```
Comment (lalli-flores) [inline src/components/WebexMeeting/WebexMeeting.jsx:126]: Let's have the default for `WebexMeeting` be [what is on `WebexMeetingControlBar` right now](https://github.com/webex/components/blob/master/src/components/WebexMeetingControlBar/WebexMeetingControlBar.jsx#L218).
Simplify `WebexMeetingControlBar` defaults to:
```
(isActive) => (
    isActive
      ? ['mute-audio', 'mute-video', 'leave-meeting']
      : ['mute-audio', 'mute-video', 'join-meeting']

File: src/components/WebexMeeting/README.md [hunk 2]
```diff
@@ -25,13 +25,20 @@ or run the following **NPM** command:
     const jsonAdapter = new WebexJSONAdapter(jsonData);
     ```
 
-2. Create a component instance by passing the meeting ID as a string. You then need to enclose it
+2. Create a component instance by passing the meeting ID as a string and an optional function. This function returns an array
```
Comment (lalli-flores) [inline src/components/WebexMeeting/README.md:28]:
```suggestion
2. Create a component instance by passing the meeting ID as a string and an optional function. This function should take a boolean parameter, which signifies whether the meeting is active and returns an array
```

File: src/components/WebexMeeting/WebexMeeting.jsx [hunk 3]
```diff
@@ -113,13 +115,19 @@ export default function WebexMeeting({
...
   logo: PropTypes.node,
   meetingID: PropTypes.string,
   style: PropTypes.shape(),
 };
 
 WebexMeeting.defaultProps = {
   className: '',
+  controls: (isActive) => (
+    isActive
+      ? ['mute-audio', 'mute-video', 'share-screen', 'member-roster', 'settings', 'leave-meeting']
```
Comment (cipak) [inline src/components/WebexMeeting/WebexMeeting.jsx:128]: these defaults don't match with the ones described in the README file

**Review by cipak — APPROVED** [review]

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**